models/README.md

```diff
@@ -80,6 +80,10 @@ engine_config:
   # vLLM keyword arguments passed when constructing the model.
   engine_kwargs:
     trust_remote_code: true
+    # Optional quantization configuration
```

Comment:
Could you remove this section from the `README`? I believe passing `awq` here makes the config file invalid because `mosaicml/mpt-7b-instruct` isn't a quantized model.

I could be misunderstanding though, so please let me know if that doesn't make sense!

Reply:
How about explicitly mentioning that this is only required for the quantized model?